ebean-core/src/main/java/io/ebeaninternal/server/core/DefaultServer.java

```diff
@@ -828,7 +828,16 @@ public void merge(Object bean, MergeOptions options, @Nullable Transaction trans
   @Override
   public <T> T mergeBeans(T bean, T existing, BeanMergeOptions options) {
     BeanDescriptor<?> desc = desc(bean.getClass());
-    return (T) desc.mergeBeans(checkEntityBean(bean), (EntityBean) existing, options);
+    EntityBean existingBean = (EntityBean) existing;
+    if (existingBean == null) {
+      existingBean = desc.createEntityBean();
+      SpiTransaction t = transactionManager.active();
+      if (t != null) {
+        SpiPersistenceContext pc = t.getPersistenceContext();
+        existingBean._ebean_getIntercept().setPersistenceContext(pc);
+      }
+    }
```

Comment:
Die "bean" kommt aus dem JSON, die "existing" Bean aus der DB.
Nun kann es sein, dass es die Bean noch nicht gibt. Dann geben wir diese in den aktuellen Persistence-Context.
Dies ist z.B dann gut, wenn 100 AclEntries im Batch gemerged werden, es aber nur 50 in der DB gibt. Die restlichen 50 müssen wir erzeugen. Damit diese "neuen" Beans effizient nachladen können, impfen wir den PC rein